README.md

```diff
@@ -58,3 +58,11 @@ Indexer was built in a way that strongly coupled it to Postgresql, and the defin
 Going forward we will continue to maintain the Indexer application, however our main focus will be enabling and optimizing a multitude of use cases through the Conduit pipeline design rather the singular Indexer pipeline.
 
 For a more detailed look at the differences between Conduit and Indexer, see [our migration guide](./docs/tutorials/IndexerMigration.md).
+
+# Known Issues
+
+## Restarting Follower Nodes
```

Comment:
"Restarting Follower Nodes Multiple Times In a Row"? 
Don't want people to think there's an issue with just restarting a follower node once, which should be a relatively common and safe thing to do.

Reply:
```suggestion
## Restarting Follower Nodes Multiple Times in a Row
```